### PR TITLE
Add pinned notes to tag pages

### PR DESCRIPTION
**Pull request by jywarren**
Follow-up to #5681 #5807 #5781 

We may want to add a test for this to screenshots test?

### CODE REVIEW

**Comment by jywarren** [comment]
```
ERROR["test_check_tag_show_page", #<Minitest::Reporters::Suite:0x00005608fa2925f8 @name="TagControllerTest">, 2.577708713999982]
 test_check_tag_show_page#TagControllerTest (2.58s)
NoMethodError:         NoMethodError: undefined method `first' for nil:NilClass
            test/functional/tag_controller_test.rb:46:in `block in <class:TagControllerTest>'
```

**Comment by jywarren** [comment]
image

**Comment by jywarren** [comment]
Testing @publiclab-mimi

**Comment by publiclab-mimi** [comment]
Yes, I got the notification!

File: app/views/notes/_notes.html.erb [hunk 1]
```diff
@@ -1,88 +1,23 @@
 <% notes = notes || @notes # accept local if present, default to instance %>
+<% pinned_nodes = pinned_nodes || @pinned_nodes || [] # accept local if present, default to instance %>
+<% tagname = tagname || false %>
 <div id="notes">
-    <div class="row">
-      <% notes.each_with_index do |node,i| %>
-        <div class="<% if @widget %>col-4 <% end %><%= node.tagnames_as_classes %> col-lg-4 clearfix node note node-nid-<%= node.id %> note-nid-<%= node.id %><% if node.status == 4 %> moderated<% end %>">
```
Comment (jywarren) [inline app/views/notes/_notes.html.erb]: We missed these lines!

**Pull request merged**